arduino-ide-extension/src/node/sketches-service-impl.ts

```diff
@@ -184,8 +184,11 @@ export class SketchesServiceImpl
   }
 
   private async sketchbookUri(): Promise<string> {
-    const { sketchDirUri } = await this.configService.getConfiguration();
-    return sketchDirUri;
+    const { config, messages } = await this.configService.config();
+    if (!config) {
+      throw new Error(messages ? messages[0] : 'cli config was undefined');
```

Comment:
must be handled in getSketches

Reply:
https://github.com/arduino/arduino-ide/pull/1774/commits/23c2791c814d0be5d04b3f1afc1b661a8605f998